client/app/utils/index.js

```diff
@@ -3,22 +3,18 @@ import axios from 'axios';
 import renderHTML from 'react-render-html';
 import { sanitize } from 'dompurify';
 
-const randomString = (): string => Math.random()
-  .toString(36)
-  .substring(2, 15)
-  + Math.random()
-    .toString(36)
-    .substring(2, 15);
+export const randomString = (): string => Math.random().toString(36).substring(2, 15)
```

Comment:
I don't think it's necessary to export these functions individually in order to be able to test them. You should be able to import the `Utils` object and call the functions as stated in the previous comment.